web/client/utils/styleparser/__tests__/CesiumStyleParser-test.js

```diff
@@ -537,5 +537,39 @@ describe('CesiumStyleParser', () => {
                     }).catch(done);
                 });
         });
+
+
+        // eslint-disable-next-line no-only-tests/no-only-tests
+        it.only('should add leader line for relative height Reference', (done) => {
+
+            const map = {};
+            const cartographic = {
+                height: 5000,
+                latitude: 0.7763247989914425,
+                longitudes: -1.8117186716869715
+            };
+            const heightReference = "relative";
+            const zValue = 1;
+            const computedHeight = {
+                none: cartographic.height,
+                relative: cartographic.height + zValue,
+                clamp: zValue
+            };
+
+            const expectedValue = Cesium.Cartesian3.fromRadiansArrayHeights([
+                cartographic.longitudes,
+                cartographic.latitude,
+                zValue,
+                cartographic.longitudes,
+                cartographic.latitude,
+                computedHeight[heightReference ?? "none"]
+            ]);
+
+            const parse = getLeaderLinePositions(map, cartographic, heightReference);
+            parse.then((res) => {
+                expect(res).toBe(expectedValue);
+                done();
+            }).catch(done());
```

Comment:
The test needs to be done with the CesiumStyleParser class see this [other test as an example](https://github.com/geosolutions-it/MapStore2/blob/7d87fd5a77e19a0a8125bc7380b1d6b445ad6e08/web/client/utils/styleparser/__tests__/CesiumStyleParser-test.js#L382-L539)